docs/smart-contract-devs/chain-integration.md

```diff
@@ -0,0 +1,28 @@
+---
+sidebar_position: 8
+sidebar_label: "⛓ Chains integration"
+---
+
+# ⛓ Chains integration
+
+## EVM-compatible chains
+
+RedStone Oracles can be integrated with EVM-compatible chains out of the box thanks to the [evm-connector](https://docs.redstone.finance/docs/smart-contract-devs/getting-started#usage). Examples of the chains tested by our team can be found [here](https://showroom.redstone.finance/).
+
+If you want to test your chain there is example contract and script which can be found [here](https://github.com/redstone-finance/redstone-showroom/tree/main/example). You need to follow two steps:
```

Comment:
is example contract and script which can be found -> is an example contract and a script here